src/auro-flight-main.js

```diff
@@ -73,10 +76,66 @@ class AuroFlightMain extends LitElement {
     return localizedTime;
   }
 
+  /**
+   * @private
+   * @param {string} station Airport code ex: SEA.
+   * @returns Mutated string.
+   */
+  readStation(station) {
+    return Array.from(station).join(' ');
+  }
+
+  /**
+   * @param {number} idx A numbered index correlated to current stop.
+   * @private
+   * @returns A comma string or an empty string.
+   */
+  addComma(idx) {
+    return idx === this.stops.length - 1 ? '' : ', ';
+  }
+
+  /**
+   * @private
+   * @returns Composed screen reader summary.
+   */
+  composeScreenReaderSummary() {
+    const hasReroute = this.reroutedDepartureStation && this.reroutedDepartureStation !== 'undefined';
+    const dayDiff = getDateDifference(this.departureTime, this.arrivalTime);
+    const daysFromDeparture = dayDiff === 1 ? 'next day' : `${dayDiff} days later`;
+    const secondToLastIndex = 2;
+    const layoverStopoverStringArray = this.stops
+      ? this.stops.map((segment, idx) => html`
+      with a ${segment.isStopover ? 'stop' : 'layover'} in ${this.readStation(segment.arrivalStation)}
+      ${segment.duration ? `, for ${segment.duration}` : ''}${this.addComma(idx)}
+      ${idx === this.stops.length - secondToLastIndex ? ' and ' : ''}`)
+      : ', nonstop';
+
+    const departureStation = this.readStation(this.departureStation);
+    const departureTime = this.convertTime(this.departureTime);
+    const arrivalStation = this.readStation(this.arrivalStation);
+    const arrivalTime = this.convertTime(this.arrivalTime);
+    let reroutedDepartureStation = '';
+    let reroutedArrivalStation = '';
+
+    if (hasReroute) {
+      reroutedDepartureStation = this.readStation(this.reroutedDepartureStation);
+      reroutedArrivalStation = this.readStation(this.reroutedArrivalStation);
+    }
+
+    return html`
+      ${!hasReroute
+        ? `Departs from ${departureStation} at ${departureTime}, arrives ${arrivalStation} at ${arrivalTime}`
+        : `Flight ${reroutedDepartureStation} to ${reroutedArrivalStation} has been re-routed.
+        The flight now departs from ${departureStation} at ${departureTime},
+        and arrives ${arrivalStation} at ${arrivalTime}`} ${dayDiff > 0 ? `, ${daysFromDeparture}` : ''}
+        ${this.stops ? ', ' : ''} ${layoverStopoverStringArray}.
+    `;
+  }
```

Comment:
NOTE for new issue: this content needs to be localized.

Reply:
@blackfalcon  We are passing in the arrival and departure datetime value as a prop into the auro-flight component. I am curious if it is it better for the localization happen outside of the auro-flight component or within?

Reply:
@shonore-aa great question. If the content is being passed INTO the element, that requires translation PRIOR to injection. But... if the content is contained WITHIN the element, that's the element's responsibility to have a localized version of the content. 

See [auro-input](https://github.com/AlaskaAirlines/auro-input/blob/main/src/i18n.js) as an example. 

For the future FUTURE the goal is that elements will subscribe to content and that will subscription will manage the localization. 

My point here is that as this element stands, anyone using a screen reader will ONLY get info back in English.

Reply:
@blackfalcon is it okay if we create a follow up Issue for handling localization?

Reply:
Yes.

Reply:
Looks like there is another issue created for this already in the auro-flight repo https://github.com/AlaskaAirlines/auro-flight/issues/41